config/initializers/webauthn.rb

```diff
@@ -0,0 +1,4 @@
+WebAuthn.configure do |config|
+  config.origin = ENV.fetch("WEBAUTHN_ORIGIN", "http://localhost:3000")
+  config.rp_name = "RubyGems.org"
```

Comment:
In guess that's the correct name? I would say "rubygems.org" personnaly but this was in the other pull request